### DISCUSSION TITLE
the "Remove Unused Import" command should remove all unused imports in the `from import` statement

### DISCUSSION THREAD

**Discussion by yvvt0379** [microsoft/pylance-release#2646] [Enhancement]
### Current Behavior
Just removes the import where the cursor is located, which requires many many repetitive operations to remove all unused imports
### Expected Behavior
Remove all unused imports in the python file.

**Comment by Chris-hughes10** [comment]
Coming to VSCode as a PyCharm user, it would be great if there was an option to remove all unused imports in a file.

**Comment by judej** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.

**Comment by heejaechang** [comment]
implemented. see https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#2022741-20-july-2022 for more detail.
